src/App.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { FC } from 'react';
-import { Footer, Header, Main } from 'components';
+import { Route, Routes } from 'react-router-dom';
+import { CatalogPage, Footer, Header, Main } from 'components';
 
 const App: FC = () => (
 	<div className="app">
 		<Header />
-		<Main />
+		<Routes>
+			<Route path="/" element={<Main />} />
+			<Route path="/CatalogPage" element={<CatalogPage title="Popular Movies" />} />
```

Comment:
no need pass title as prop. move inside page

Reply:
This page I wil reuse for another categories of movies, like popular, now playing and other, so we need do it